packages/modules/pricing/integration-tests/__tests__/services/pricing-module/price-list.spec.ts

```diff
@@ -809,6 +858,24 @@ moduleIntegrationTestRunner({
               price_list_rules: [],
             })
           )
+
+          expect(eventBusEmitSpy.mock.calls[0][0]).toHaveLength(2)
```

Comment:
todo: l 796 add `jest.clearAllMocks()` to not be bothered if any of the previous method to populate some data emit some events. You can usually do that just before the method you want to check the events from. I let you validate where you need it in all the tests :D

Reply:
Aren't we doing this in `beforeEach`?